pennylane/beta/tapes/tape.py

```diff
@@ -0,0 +1,776 @@
+
+    @trainable_params.setter
+    def trainable_params(self, param_indices):
+        """Store the indices of parameters that support differentiability.
+
+        Args:
+            param_indices (set[int]): parameter indices
+        """
+        if any(not isinstance(i, int) or i < 0 for i in param_indices):
+            raise ValueError("Argument indices must be positive integers.")
+
+        if any(i > len(self._par_info) for i in param_indices):
+            raise ValueError(f"Tape has at most {self.num_params} parameters.")
+
+        self._trainable_params = param_indices
+
+    @property
+    def operations(self):
+        """Returns the operations on the quantum tape.
+
+        Returns:
+            list[.Operation]: list of recorded quantum operations
+
+        **Example**
+
+        .. code-block:: python
+
+            from pennylane.beta.tapes import QuantumTape
+            from pennylane.beta.queuing import expval, var, sample, probs
+
+            with QuantumTape() as tape:
+                qml.RX(0.432, wires=0)
+                qml.RY(0.543, wires=0)
+                qml.CNOT(wires=[0, 'a'])
+                qml.RX(0.133, wires='a')
+                expval(qml.PauliZ(wires=[0]))
+
+        >>> tape.operations
+        [RX(0.432, wires=[0]), RY(0.543, wires=[0]), CNOT(wires=[0, 'a']), RX(0.133, wires=['a'])]
+        """
+        return self._prep + self._ops
+
+    @property
+    def observables(self):
+        """Returns the observables on the quantum tape.
+
+        Returns:
+            list[.Observable]: list of recorded quantum operations
+
+        **Example**
+
+        .. code-block:: python
+
+            from pennylane.beta.tapes import QuantumTape
+            from pennylane.beta.queuing import expval, var, sample, probs
+
+            with QuantumTape() as tape:
+                qml.RX(0.432, wires=0)
+                qml.RY(0.543, wires=0)
+                qml.CNOT(wires=[0, 'a'])
+                qml.RX(0.133, wires='a')
+                expval(qml.PauliZ(wires=[0]))
+
+        >>> tape.operations
+        [expval(PauliZ(wires=[0]))]
+        """
+        return [m[1] for m in self._obs]
+
+    @property
+    def measurements(self):
+        """Returns the measurements on the quantum tape.
+
+        Returns:
+            list[.MeasurementProcess]: list of recorded measurement processess
+
+        **Example**
+
+        .. code-block:: python
+
+            from pennylane.beta.tapes import QuantumTape
+            from pennylane.beta.queuing import expval, var, sample, probs
+
+            with QuantumTape() as tape:
+                qml.RX(0.432, wires=0)
+                qml.RY(0.543, wires=0)
+                qml.CNOT(wires=[0, 'a'])
+                qml.RX(0.133, wires='a')
+                expval(qml.PauliZ(wires=[0]))
+
+        >>> tape.measurements
+        [<pennylane.beta.queuing.measure.MeasurementProcess object at 0x7f10b2150c10>]
+        """
+        return [m[0] for m in self._obs]
```

Comment:
:thinking: This is one where the difference between `measurements` and `observables` is not too clear to me

Reply:
Should be fixed now!